resources/profiles/MagicMaker/machine/MM BoneKing 0.4 nozzle.json

```diff
@@ -1,26 +1,27 @@
 {
 	"type": "machine",
 	"setting_id": "GM001",
-	"name": "MM BoneKing 0.4 nozzle",
+	"name": "MM slb 0.4 nozzle",

```

Comment:
Isn't there already a MM slb 0.4 nozzle file?
seem you accidentally copied that file content here. Please double check.